tests/pactHelper.js

```diff
@@ -270,10 +270,16 @@ async function getCapabilitiesInteraction (
       query: { format: 'json' },
       headers: { authorization: getAuthHeaders(user, password) }
     })
+    // [oCIS] ocis returns `text/plain` content-type
+    // not `application/json` like oc10 when requesting with `?format=json`
+    // https://github.com/owncloud/ocis/issues/1779
```

Comment:
Maybe, you should consider using multiline comments? :thinking: 
Other than that, everything LGTM :+1:

Reply:
do you mean `/* ... */`?

Reply:
yes